server/src/main/java/com/epam/aidial/core/server/util/SpanUtil.java

```diff
@@ -12,7 +12,15 @@ public class SpanUtil {
     private static final List<String> GROUPS = List.of("id", "bucket", "path");
 
     public static void updateName(Pattern pathPattern, String path, String httpMethod) {
-        String spanName = RegexUtil.replaceNamedGroups(pathPattern, path, GROUPS);
-        Span.current().updateName(httpMethod + " " + spanName);
+        String httpPath = RegexUtil.replaceNamedGroups(pathPattern, path, GROUPS);
```

Comment:
We can do code re-factoring and move the methods to SelectorController class.

So the class is no longer needed in this case.

Reply:
Will be fixed in scope of a separate issue